### PR TITLE
eslint-setup

### PR DESCRIPTION
**Pull request by facundoveliz**

### CODE REVIEW

**Review by martinmolina1988 — APPROVED** [review]

**Review by patriciodfernandez — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+    'react/prop-types': 'off',
+  },
```
Comment (dvelezroman) [inline .eslintrc.js:27]: aniadir 
- no console.logs
- no semicolons
- comillas simples
- react hooks deps

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+    'react/prop-types': 'off',
```
Comment (dvelezroman) [inline .eslintrc.js:26]: esto deberia ir on

**Review by dvelezroman — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (dvelezroman) [reply .eslintrc.js:33]: `"react/jsx-props-no-spreading": "off",
    "no-console": "error",
    "semi": "off",
    "no-param-reassign": "off",
    "linebreak-style" : 0,
    "implicit-arrow-linebreak": "off"`

**Comment by dvelezroman** [comment]
crear un archivo 
.eslintignore 
`build
node_modules
.github
ServiceWorker.js
.github/
.husky/
package-lock.json
*.test.js`

**Review by dvelezroman — DISMISSED** [review]

**Pull request merged**